test/bntest.c

```diff
@@ -185,6 +185,7 @@ static int test_sub()
 {
     BIGNUM *a, *b, *c;
     int i;
+    int st = 0;
 
     a = BN_new();
```

Comment:
Do you mean for all the similar statements in this file like `a = BN_new();`. we should check them by `if (a == NULL) goto err;`